model_api/python/openvino/model_api/models/classification.py

```diff
@@ -32,6 +34,15 @@ def __init__(self, inference_adapter, configuration=None, preload=False):
         if self.path_to_labels:
             self.labels = self._load_labels(self.path_to_labels)
         self.out_layer_names = [self._get_output()]
+
+        if self.hierarchical:
+            self.embedded_processing = True
+            self.hierarchical_info = json.loads(self.hierarchical_config)
```

Comment:
Can you throw `if self.hierarchical and not self.hierarchical_config` after my C++ comment